src/corehost/cli/fxr/sdk_info.h

```diff
@@ -8,18 +8,22 @@
 
 struct sdk_info
 {
-    sdk_info(pal::string_t path, fx_ver_t version)
-        : path(path)
-        , version(version) { }
+    sdk_info(pal::string_t base_path, pal::string_t full_path, fx_ver_t version, int32_t hive_depth)
```

Comment:
nit: the original code here should probably be `const &` for the string and version (pass in by ref then assign fields by value) to avoid an extra copy

Reply:
Fixed.